frontend/apps/thunder-develop/package.json

```diff
@@ -34,20 +34,41 @@
   "dependencies": {
     "@asgardeo/react": "catalog:",
     "@asgardeo/react-router": "catalog:",
+    "@dnd-kit/abstract": "0.1.21",
+    "@dnd-kit/collision": "0.1.21",
+    "@dnd-kit/core": "6.3.1",
+    "@dnd-kit/dom": "0.1.21",
+    "@dnd-kit/helpers": "0.1.21",
+    "@dnd-kit/react": "0.1.21",
     "@emotion/react": "catalog:",
     "@emotion/styled": "catalog:",
     "@hookform/resolvers": "catalog:",
+    "@lexical/html": "0.38.2",
+    "@lexical/link": "0.38.2",
+    "@lexical/react": "0.38.2",
+    "@lexical/rich-text": "0.38.2",
+    "@lexical/selection": "0.38.2",
+    "@lexical/utils": "0.38.2",
     "@mui/material": "catalog:",
     "@tanstack/react-query": "catalog:",
     "@tanstack/react-query-devtools": "catalog:",
     "@thunder/commons-contexts": "workspace:^",
     "@thunder/i18n": "workspace:^",
     "@wso2/oxygen-ui": "catalog:",
     "@wso2/oxygen-ui-icons-react": "catalog:",
+    "@xyflow/react": "catalog:",
+    "@xyflow/system": "^0.0.72",
+    "classnames": "2.5.1",
     "clsx": "catalog:",
+    "dagre": "0.8.5",
     "dayjs": "catalog:",
+    "dompurify": "3.3.0",
+    "elkjs": "^0.11.0",
+    "html-react-parser": "5.2.10",
     "human-id": "4.1.2",
     "i18next": "catalog:",
+    "lexical": "0.38.2",
+    "lodash-es": "catalog:",
```

Comment:
Lets see if we can use the micro modules from `lodash-es` instead of the whole package to avoid usage of already available JS utils like `filter`, `find` etc.

ex: https://www.npmjs.com/package/lodash.clonedeep